### ISSUE TITLE
[Bug] Belt Stoppers stopped working

### ISSUE THREAD

**Issue by mooleshacat** [DellAquila/PowerControlRoom#6]
Not sure why, but the belt stoppers stopped working all of the sudden...

The switch powering them is turned on, but the splitter still shows red lights and does not start allowing items to pass.

The switch worked perfectly fine before.

I suspect an update to the game broke the belt stopper.

**Update:**
I looked back at my other monitor where the game was running just now, and the light was green and items were able to pass...

So I threw the switch back to off to see if it would stop the belt, and the belt just keeps going, the only way to stop it is to delete the belt going into or out of the belt stopper.

Basically it seems the entire funtionality (start and stop) of the belt stopper does not work.

**Comment by mooleshacat** [comment]
Update:

I looked back at my other monitor where the game was running just now, and the light was green and items were able to pass...

So I threw the switch back to off to see if it would stop the belt, and the belt just keeps going, the only way to stop it is to delete the belt going into or out of the belt stopper.

Basically it seems the entire funtionality (start and stop) of the belt stopper does not work.

**Comment by mooleshacat** [comment]
For some reason, without any update to the game or mod, the belt stopper works now.

There is something causing the belt stopper to intermittently stop working.

**Comment by mooleshacat** [comment]
It appears to be working again. I think maybe it has to do with my battery storage. If I add a bunch of new batteries that need to recharge, would they take up the power needed to switch the stopper on/off?

It could be the battery storage.
